lib/supply/client.rb

```diff
@@ -211,10 +248,16 @@ def upload_apk_to_track_with_rollout(path_to_apk, track, rollout)
 
       raise result_upload.error_message.red if result_upload.error?
 
+      return result_upload.data.versionCode
+    end
+
+    def update_track(track, rollout, apk_version_code)
+      ensure_active_edit!
+
       track_body = {
         'track' => track,
         'userFraction' => rollout,
-        'versionCodes' => [result_upload.data.versionCode]
+        'versionCodes' => [apk_version_code]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.